wg-api/best-practices.md

```diff
@@ -32,6 +32,13 @@ function whatever(opts: { a: string, b?: boolean }) { /* ... */ }
 
 See https://w3ctag.github.io/design-principles/#prefer-dict-to-bool for more details.
 
+### Should the configured options be made extensible to third-party libraries?
+
+If an API accepts configuring options, should it provide the ability to append to rather than replace those options?
+Would third-party libraries require special attention to API usage to avoid clobbering configured options?
```

Comment:
I think this might actually apply to more than just configuring options, though that's the most obvious case. Here's some suggested tweaks to prompt us to think more broadly about how loosely coupled code will interact with an API. Feel free to edit further (or ignore) if you don't like my wording 😄 

```suggestion
### How will third-party libraries interact with this API?

When designing an API, consider that an app might have custom code and third-party libraries that both interact with that API. Can the API be designed so that multiple callers don't interfere with each other?

If an API accepts configuring options, should it provide the ability to append to rather than replace those options?
Can third-party libraries use the API without knowing what app-specific code is also using the API?
```

Reply:
Good point, and I like your phrasing better 👍